src/main/java/hudson/plugins/git/extensions/impl/PruneStaleTag.java

```diff
@@ -68,6 +73,15 @@ public PruneStaleTag(boolean pruneTags) {
         this.pruneTags = pruneTags;
     }
 
+    /**
+     * Needed for pipeline syntax generator.
+     * 
```

Comment:
Please remove the trailing whitespace from this comment.
```suggestion
     *
```